### PR TITLE
posts loader

### PR DESCRIPTION
**Pull request by bladicito**

### CODE REVIEW

File: app/api/posts/route.ts [hunk 1]
```diff
@@ -4,15 +4,29 @@ import { NextRequest, NextResponse } from 'next/server';
...
+    limit: string;
+  };
+}
+
 export const GET = async (request: NextRequest): Promise<Response> => {
+  const searchParams = request.nextUrl.searchParams;
+  const limit = searchParams.get('limit');
+  const offset = searchParams.get('offset');
+
+  const limit1 = limit ? parseInt(limit, 10) : config.feed.defaultAmount;
```
Comment (lehmi11) [inline app/api/posts/route.ts:20]: could you rename those variables? 
My suggestion:
- line 17: limitQueryParam 
- line 18: offsetQueryParam 
- line 20: limit
- line 21: offset

File: app/layout.tsx [hunk 2]
```diff
@@ -27,22 +28,27 @@ export default function RootLayout({
...
 }) {
+  const mainWrapperCss =
+    'main-wrapper w-full flex flex-col h-screen min-h-screen overflow-y-hidden';
+  const headerCss = 'header bg-violet-600 py-3';
+  const contentCss = 'content bg-slate-100 px-10 lg:px-0 grow overflow-y-auto';
+
   return (
     <html>
-      <body className={inter.className}>
+      <body className={`${inter.className} `}>
```
Comment (lehmi11) [inline app/layout.tsx:38]: Why do you need to turn this className into a template string?

File: app/page.tsx [hunk 3]
```diff
@@ -2,35 +2,41 @@ import WelcomeTexts from '@/components/welcome-texts/WelcomeTexts';
...
       <div className="max-w-4xl mx-auto">
         {session && (
           <div>
-            <PostEditorPlaceholder />
             <PostEditor isFeedPage={true} />
           </div>
         )}
 
         <div className="max-w-4xl mr-auto ml-auto">
+          {/*server side rendered*/}
```
Comment (lehmi11) [inline app/page.tsx:31]: Do we need these comments if those components are client / server side rendered? If someone change those component's rendering strategy, those comments could be irritating.

File: components/posts-loader/PostsLoader.tsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    (node: any) => {
+      if (isLoading) return;
+      if (observer.current) {
+        // @ts-ignore
+        observer.current.disconnect();
+      }
+      // @ts-ignore
+      observer.current = new IntersectionObserver((entries) => {
+        if (entries[0].isIntersecting) {
+          // console.log('reached to the last post');
```
Comment (lehmi11) [inline components/posts-loader/PostsLoader.tsx:36]: there is a leftover ;-)

File: services/Mumble/index.ts [hunk 5]
```diff
@@ -63,8 +63,13 @@ export class MumbleService {
       }
 
       const response = await fetch(`${this.baseUrl}/${path}`, options);
+
+      // all but 200s are going to be rejected?
```
Comment (lehmi11) [inline services/Mumble/index.ts:67]: Who is this question for? Perhaps we could create an issue and discuss it

File: app/api/posts/route.ts (hunk 1 above)
Comment (bladicito) [reply app/api/posts/route.ts:20]: in the profile pr this part gets improved. I will adapt it here too. 👍🏽

File: app/layout.tsx (hunk 2 above)
Comment (bladicito) [reply app/layout.tsx:38]: i was adding a class indicator, which is not the case anymore. I will correct it 👍🏽

File: services/Mumble/index.ts (hunk 5 above)
Comment (bladicito) [reply services/Mumble/index.ts:67]: for me. I will check at this a bit and if have still thoughts about it, we can discuss it and open an issue. 👍🏽 
For the moment I will remove it.

**Review by lehmi11 — APPROVED** [review]

**Pull request merged**